.ci/do_release.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash -Exel
+
+echo "**********************************"
+echo "Starting do_release.sh script..."
+echo "**********************************"
+
+set -o pipefail
+
+print_help() {
+    set +xv  
+    echo -e "\n\n"
+    echo "--------------------------------------------------"
+    echo "Usage: release_folder=<release folder> release_tag=<release tag> [revision=<revision>] [do_release=<true|false>] $0"
+    echo "       Where release folder is a path to NFS folder to copy the package into"
+    echo "       Where release tag is a git tag to release (must be already tagged in the git repo)"
+    echo "       Where revision is a number postfix to add the to package indicating which version of the tag it is - OPTIONAL, default value 1"
+    echo "       Where do_release is a boolean value indicating if the script will copy the created package into the release folder location - OPTIONAL, default value false"
+    exit 1
+}
+
+if [ -z "${release_folder}" ]; then
+    echo "ERROR: 'release_folder' was not set."
+    print_help
+fi
+
+if [ ! -e "${release_folder}" ] || [ ! -d "${release_folder}" ]; then
+    echo "ERROR: [${release_folder}] directory doesn't exist."
+    print_help
+fi
+
+if [ -z "${release_tag}" ]; then
+    echo "ERROR: 'release_tag' was not set."
+    print_help
+fi
+
+if [ -z "${revision}" ]; then
+    echo "WARN: 'revision' was not set, defaulting to 1"
+    revision=1
+fi
+
+if [ -z "${do_release}" ]; then
+    echo "WARN: 'do_release' was not set, defaulting to false (package will not be release)"
+    do_release=false
+fi
+
+env PRJ_RELEASE="${revision}" contrib/build_pkg.sh -s
+
+MAJOR_VERSION=$(grep -e "define(\[prj_ver_major\]" configure.ac | awk '{ printf $2 };' | sed  's/)//g')
+MINOR_VERSION=$(grep -e "define(\[prj_ver_minor\]" configure.ac | awk '{ printf $2 };' | sed  's/)//g')
+REVISION_VERSION=$(grep -e "define(\[prj_ver_revision\]" configure.ac | awk '{ printf $2 };' | sed  's/)//g')
+configure_ac_version="${MAJOR_VERSION}.${MINOR_VERSION}.${REVISION_VERSION}"
+echo "FULL_VERSION from configure.ac: [${configure_ac_version}]"
+
+if [[ "${release_tag}" != "${configure_ac_version}" ]]; then
+    echo "ERROR: FULL_VERSION: ${configure_ac_version} from configure.ac doesn't match tag: ${release_tag} provided! Exit"
+    exit 1
+fi
+
+if [ "${do_release}" = true ] ; then
+    echo "do_release is set to true, will release package into ${release_folder}/libvma_v_${release_tag}-${revision}"
+
+    cd pkg/packages || { echo "pkg folder is missing, exiting..."; exit 1; }
+    pkg_name=$(ls -1 libvma-"${release_tag}"-"${revision}".src.rpm)
+    DST_DIR=${release_folder}/libvma_v_${release_tag}-${revision}
```

Comment:
looking at the release folder and this is not the path / names convention

Existing one today:
```
$ ls -ltr /auto/mswg/release/vma/vma_v_9.8.60-0/src/
-rw-r--r-- 1 danielli labop 1600185 Feb 28 16:08 libvma-9.8.60-1.src.rpm
```

and yours is (look at the folder name and the missing src folder
```
$ ls -ltr /auto/mswg/release/vma/tmp/libvma_v_9.8.60-0
-rw-r--r-- 1 swx-jenkins labop 1602856 Jul 31 15:25 libvma-9.8.60-0.src.rpm
```